README.md

```diff
@@ -1 +1,23 @@
-# parasoft-demo-app-mobile
\ No newline at end of file
+# parasoft-demo-app-mobile
+The Parasoft Demo Application is an example Android application. The application is configurable, and is used to demonstrate functionality in a variety of Parasoft tools.
+
+## Getting Started
+### Create emulator
```

Comment:
Explain to user that they can either use emulator comes with Android Studio or thrid party emulator like BlueStacks. 
Recommend an emulator which best support Android 13 system which SDK level of API 33 (I just updated from 32 to 33) instead of the minimum.
Then explain minimum solution.